src/content/documentation/publish/site-permission-add-on.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: full-width
+title: Site permission add-on
+permalink: /documentation/publish/site-permission-add-on/
+topic: Publish
+tags: [add-on, distribution, publication, signing, installation]
+contributors: rebloor
+last_updated_by: rebloor
+date: 2022-02-17
+---
+
+<!-- Page Hero Banner -->
+
+{% capture page_hero_banner_content %}
+
+# Site Permission Add-ons
+
+{% endcapture %}
+{% include modules/page-hero.liquid
+	content: page_hero_banner_content
+%}
+
+<!-- END: Page Hero Banner -->
+
+<!-- Single Column Body Module -->
+
+{% capture content %}
+
+::: note
+This is an experimental feature.
+:::
+
+
+If your site depends on an add-on-gated feature, your users need to install an add-on before using your service. This means that to set up your service, you need to:
+
+- Request a site permission add-on from addons.mozilla.org (AMO).
+- Make the site permission add-on available to download from your website.
+- Ask your users to install the add-on before using your service.
+
+Once the add-on is installed, your service gains access to the restricted API. This page provides a step-by-step guide to the process.
+
+## Obtain your site permission add-on
+
+To obtain the file containing your site permission add-on, do this:
+
+- If you don't have a Firefox account, [create one](https://support.mozilla.org/en-US/kb/how-do-i-set-sync-my-computer#w_create-a-firefox-account) in the Firefox browser using the **Sync and save data** option on the Firefox menu.
+- Visit the [AMO site permission generator](https://addons.mozilla.org/en-US/developers/site_permission_generator/).
+- If you aren't registered on addons.mozilla.org, you are prompted to:
+  - complete your profile.
+  - accept the Firefox Add-on Distribution Agreement.
+- In the **Generate Site Permission Add-on** screen, enter the URL of the site that requires API access and select the APIs it wants access to. If you have more than one site that needs access to these APIs, you need to create one add-on for each site.
```

Comment:
```suggestion
- In the **Generate Site Permission Add-on** screen, enter the URL of the site that requires API access and select the APIs it wants access to. If you have more than one site that needs access to these APIs, you need to create one add-on for each site. The separation is by origin, therefore subdomains will require a separate add-on.
```

I was thinking about this as well. I felt using "origin" instead of "site" may be harder to understand for some and went with Richard's more generic language. I've added a suggestion, @tublitzed  does that sound good to you?

Reply:
Yup, thanks!